dashboard/src/components/common/CreateTicket/CreateTicket.tsx

```diff
@@ -0,0 +1,250 @@
+                )}
+              >
+                {projects.map((proj) => (
+                  <Option
+                    value={proj.projectSubdomain}
+                    key={proj.projectSubdomain}
+                  >
+                    {proj.projectName} ({proj.projectSubdomain})
+                  </Option>
+                ))}
+              </Select>
+
+              <StyledInput
+                {...register('subject')}
+                id="subject"
+                label="Subject"
+                placeholder="Subject"
+                fullWidth
+                autoFocus
+                inputProps={{ min: 2, max: 128 }}
+                error={!!formState.errors.subject}
+                helperText={formState.errors.subject?.message}
+                onChange={(e) => {
+                  setSubject(e.target.value);
+                }}
+                value={subject}
+              />
+
+              <Select
+                {...register('severity')}
+                id="severity"
+                label="Severity"
+                placeholder="Severity"
+                slotProps={{
+                  root: { className: 'grid grid-flow-col gap-1' },
+                }}
+                onChange={(_, value) => {
+                  setSeverity(value as string);
+                }}
+                value={severity}
+                renderValue={(option) => (
+                  <span className="inline-grid grid-flow-col items-center gap-2">
+                    {option?.label}
+                  </span>
+                )}
+              >
+                <Option value="low" key="low">
+                  Low
+                </Option>
+                <Option value="medium" key="medium">
+                  Medium
+                </Option>
+                <Option value="high" key="high">
+                  High
+                </Option>
+                <Option value="urgent" key="urgent">
+                  Urgent
+                </Option>
+              </Select>
+
+              <StyledInput
+                {...register('description')}
+                id="description"
+                label="Description"
+                placeholder="Description"
+                fullWidth
+                multiline
+                inputProps={{ min: 2, max: 128 }}
+                error={!!formState.errors.description}
+                helperText={formState.errors.description?.message}
+                onChange={(e) => {
+                  setDescription(e.target.value);
+                }}
+                value={description}
+              />
+
+              <Button
+                variant="outlined"
+                color="secondary"
+                className="hover:!bg-white hover:!bg-opacity-10 focus:ring-0"
+                size="large"
+                type="submit"
+                disabled={formState.isSubmitting}
+                loading={formState.isSubmitting}
+                onClick={handleSubmit}
```

Comment:
No need to have this here because the type is `submit` here. This will probably fire twice this way